src/phone-number-input/phone-number-input.tsx

```diff
@@ -118,6 +120,40 @@ export const PhoneNumberInput = ({
         };
     };
 
+    const getAddonProps = (): AddonProps<CountryValue, string> => {
+        if (fixedCountry) {
+            return {
+                type: "label",
+                attributes: {
+                    value: selectedCountryCode,
+                } as LabelAddon,
```

Comment:
no need to cast here?

Reply:
We need to else I will be able to pass list addon props